### PR TITLE
Expect different formatting of PM2.5

### PR DESCRIPTION
**Pull request by dolugen**
#233

### CODE REVIEW

**Comment by dolugen** [comment]
I think I'm gonna go ahead and merge this.

**Comment by jflasher** [comment]
@dolugen please don't merge until issues raised in comments are addressed, thanks!

**Comment by dolugen** [comment]
By "issues raised in comments", do you mean Travis checks? Or something else?

**Comment by jflasher** [comment]
https://github.com/openaq/openaq-fetch/pull/238#discussion_r100393059

**Comment by dolugen** [comment]
The link's not going anywhere for me.

**Comment by jflasher** [comment]
Haha, looks like I was one button click away from greatness. Apparently I never actually clicked the 'submit' button but it was still showing for me. Sorry! Should be a comment showing up now.

Also, please don't merge at the moment since we're blocked with some of the EEA work on `develop`. I just need to test that a bit more and then will merge this in as well.

**Comment by dolugen** [comment]
Ok, I see it now.

File: adapters/cpcb.js [hunk 1]
```diff
@@ -162,10 +162,14 @@ const renameParameter = function (parameter) {
       return 'co';
     case 'PM2.5':
       return 'pm25';
+    case 'PM 2.5':
```
Comment (dolugen) [inline adapters/cpcb.js]: I've updated this as requested.

**Comment by jflasher** [comment]
:boom: thanks. Will merge this once EEA stuff is cleared up

**Comment by dolugen** [comment]
Ok, great.

**Pull request merged**